productcat/src/main/java/com/visenze/productcat/android/api/impl/SearchOperationsImpl.java

```diff
@@ -60,6 +61,27 @@ public void imageSearch(ImageSearchParams params, final ProductCat.ResultListene
         imageSearch(params, resultListener, PRODUCT_SUMMARY_SEARCH);
     }
 
+    @Override
+    public void imageSearchResultPage(ImageSearchParams params, final ProductCat.ResultListener resultListener) {
+        byte[] imageBytes = null;
```

Comment:
the image validation is exactly the same as imageSearch method.
Can you put in a common validation method? Thanks.

Reply:
the only difference is the endpoint, should refactor for 2 methods to call the same base method with new parameter for endpoint

Reply:
changed already.